libr/debug/p/native/xnu/xnu_debug.c

```diff
@@ -516,53 +516,12 @@ int xnu_get_vmmap_entries_for_pid (pid_t pid) {
 	return n;
 }
 
-static int get_bits () {
```

Comment:
`git grep SAME_BITNESS .` doesn't return anything except this line in radare2 source code. And since it is defined in a .c file I think it's used(and should be  used) only there. Is it used in other repositories?